content/v2.0/reference/api/python-cl-guide.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Python client library
+description: >
+  Use the Python client library to interact with InfluxDB
+weight: 103
+menu:
+  v2_0_ref:
+    name: Python client library
+    parent: InfluxDB v2 API
+v2.0/tags: [client libraries]
+---
+
+Use the [InfluxDB Python client libary](https://github.com/influxdata/influxdb-client-python) to integrate InfluxDB into Python scripts and applications.
+
+This guide presumes some familiarity with Python and InfluxDB.
+If you haven't, go ahead and read the [getting started](/v2.0/get-started/) guide.
+
+## Before you begin
+
+1. Install the InfluxDB Python library by running
+
+    ```sh
+    pip install influxdb-client
+    ```
+
+2. Ensure that InfluxDB is running.
+
+## Writing data to InfluxDB with Python
+
+We are going to write some data in [line protocol](/v2.0/reference/syntax/line-protocol/) using the Python library.
+
+The first line of our program imports the InfluxDB library:
+
+```python
+import influxdb_client
+```
```

Comment:
In your Python program, import the InfluxDB client library and use it to write data to InfluxDB.